imageio/core/functions.py

```diff
@@ -76,12 +76,55 @@
 
 from __future__ import absolute_import, print_function, division
 
+from numbers import Number
+import re
+
 import numpy as np
 
 from . import Request, RETURN_BYTES
 from .. import formats
 
 
+MEMTEST_DEFAULT_MIM = "256MB"
+MEMTEST_DEFAULT_MVOL = "1GB"
+
+
+mem_re = re.compile(r"^(\d+\.?\d*)\s*([kKMGTPEZY]?i?[bB])?$")
+sizes = {"": 1, None: 1}
+for i, si in enumerate([""] + list("kMGTPEZY")):
```

Comment:
You lost me after "P": are "EB", "ZB" and "YB" common memory size units?

Reply:
Ah, here's an explanation: https://en.wikipedia.org/wiki/Units_of_information

Reply:
They're not common, but if we can add 3 characters and future proof the code for the next 100 years it seems a shame not to ;) SI units aren't changing.

Reply:
haha, agreed